doc/content/getting-started/installation/configuration/_index.md

```diff
@@ -141,3 +143,56 @@ Below is an example `ttn-lw-stack-docker.yml` file for the Enterprise stack:
 {{< /highlight >}}
 
 > NOTE: Make note of the client secret, as it will be needed again when initializing {{% tts %}}.
+
+## Localhost
+
+If you are configuring and running {{% tts %}} on a local machine with no public IP or DNS address, some special considerations must be made. `localhost` has a different meaning on your machine than inside the Docker container where {{% tts %}} runs, which can cause problems if you use `localhost` in your configuration.
+
+`localhost` addresses on your machine will resolve to your machine, while `localhost` inside the Docker container will resolve inside the Docker container. In `docker-compose.yml`, we forward requests on ports 80 and 443 to ports 1885 and 8885 inside the container, so visiting `localhost` (which is really `localhost:80`) on your machine actually takes you to `localhost:1885` within the container.
+
+If you configure your `is.oauth.ui.canonical-url` as `localhost`, this causes `Token Exchange Refused` errors when you try to log in to the console, because an authorization request generated from within the container will not be redirected to port 1885 or 8885, where {{% tts %}} is listening.
```

Comment:
```suggestion
If you configure your `is.oauth.ui.canonical-url` as `localhost`, this causes **Token Exchange Refused** errors when you try to log in to {{% tts %}} Console, because an authorization request generated from within the container will not be redirected to port 1885 or 8885, where {{% tts %}} is listening.
```

Reply:
Not addressed